ru/turing/courses/lesson5/Student.java

```diff
@@ -0,0 +1,158 @@
+package ru.turing.courses.lesson5;
+
+import java.util.Comparator;
+import java.util.Objects;
+
+/**
+ * Класс Student с тремя полями: id, name, phone
+ */
+public class Student {
+    private Long id;
+    private String name;
+    private String phone;
+
+    public Student(Long id, String name, String phone) {
+        this.id = id;
+        this.name = name;
+        this.phone = phone;
+    }
+
+    public Long getId() {
+        return id;
+    }
+
+    public void setId(Long id) {
+        this.id = id;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public void setName(String name) {
+        this.name = name;
+    }
+
+    public String getPhone() {
+        return phone;
+    }
+
+    public void setPhone(String phone) {
+        this.phone = phone;
+    }
+
+    /**
+     * сравнение 2-х студентов по полю name
+     */
+    public static Comparator<Student> COMPARE_BY_NAME = new Comparator<Student>() {
+        @Override
+        public int compare(Student o1, Student o2) {
+            return o1.name.compareTo(o2.name);
+        }
+    };
+
+    /**
+     * сравнение 2-х студентов по полю id
+     */
+    public static Comparator<Student> COMPARE_BY_ID = new Comparator<Student>() {
+        @Override
+        public int compare(Student o1, Student o2) {
+            return o1.id.compareTo(o2.id);
+        }
+    };
+
+    /**
+     * сравнение 2-х студентов по полю phone
+     */
+    public static Comparator<Student> COMPARE_BY_PHONE = new Comparator<Student>() {
+        @Override
+        public int compare(Student o1, Student o2) {
+            return o1.phone.compareTo(o2.phone);
+        }
+    };
+
+    /**
+     * перестановка 2-х элементов с индексами first, second массива String[]
+     * @param array
+     * @param first
+     * @param second
+     */
+    private static void toSwap( String[] array, int first, int second){
```

Comment:
```suggestion
    private static void swap( String[] array, int first, int second){
```
Так по неймингу корректнее будет